tasks/rubies.yml

```diff
@@ -1,17 +1,13 @@
 ---
 
-- name: Detect if rubies are installed
-  command: '{{ rvm1_rvm }} {{ item }} do true'
+- name: Detect rubies
+  command: '{{ rvm1_rvm }} list strings'
+  register: rvm_list_strings
```

Comment:
I believe this point fails with `--check` enabled. I don't think it is a huge deal at this point though, because other parts will probably also break or output incorrect stuff (e.g. no changes where changes should be reported), so I think we can leave with that for now.